### ISSUE TITLE
.NET Core February 2021 Update - 5.0.3, 3.1.12, 2.1.25

### ISSUE THREAD

**Issue by rbhanda** [dotnet/core#5932]
Release Notes 
[5.0.3](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0.3/5.0.3.md)
[3.1.12](https://github.com/dotnet/core/blob/master/release-notes/3.1/3.1.12/3.1.12.md)
[2.1.25](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1.25/2.1.25.md)

Please report any issues you find with .NET 5.0.3, .NET Core 3.1.12 or .NET Core 2.1.25, either responding to this issue, creating a new issue or creating a new issue in one of the following repos:

* [dotnet/aspnetcore](https://github.com/dotnet/aspnetcore/issues) - for ASP.NET Core questions
* [dotnet/efcore](https://github.com/dotnet/efcore/issues) - for Entity Framework Core questions
* [dotnet/sdk](https://github.com/dotnet/sdk/issues) - for CLI tools and questions
* [dotnet/runtime](https://github.com/dotnet/runtime/issues) - for runtime, API, or installer issues and questions
* [dotnet/winforms](https://github.com/dotnet/winforms/issues) - for WinForms issues
* [dotnet/wpf](https://github.com/dotnet/wpf/issues) - for WPF issues
* [nuget/home](https://github.com/nuget/home/issues) - for NuGet questions and issues

-----
Status of availability of SDK on Microsoft's Linux feeds: 

| Distro | 5.0.3 | 3.1.12 | 2.1.25 
|--|--|--|--|
| Ubuntu 16.04   | ✅  | ✅  | ✅  |
| Ubuntu 18.04    | ✅  | ✅  | ✅  |
| Ubuntu 20.04   | ✅  | ✅  | ✅  |
| Ubuntu 20.10   | ✅  | ✅  | ✅  |
| Centos 7    | ✅  | ✅  | ✅  |
| Debian 9     | ✅  | ✅  | ✅  |
| Debian 10      | ✅  | ✅  | ✅  |
| Fedora 32    | ✅  |  ✅ | ✅  |
| Fedora 33    | ✅  | ✅  | ✅  |
| OpenSUSE 15   | ✅  | ✅  | ✅  |
| SLES 12      | ✅  | ✅  | ✅  |
| SLES15    | ✅  | ✅  | ✅  |

Note: This list refers to the Microsoft-provisioned feeds (packages.microsoft.com) and does not in any way represent direct availability in distros (eg RHEL, Fedora).

### Known Issues
If there are any issues with this release we will track them here and check issues off as they're resolved. See the linked issues for details on progress and resolution details.

- [x] Unable to install .NET Core 3.1.12 on Fedora 32. Package https://packages.microsoft.com/fedora/32/prod/dotnet-sdk-3.1.406-x64.rpm is corrupt, we will re-publish and update once it is done.

**Comment by WeihanLi** [comment]
Could we keep the same nuget package version, seemed it's the same version when netcore3.1 release, but some are different in .NET5, for example:

the latest `Microsoft.Extensions.Http`/`Microsoft.Extensions.Options` is `5.0.0`
the latest `Microsoft.Extensions.Http.Polly` is `5.0.1`
the latest `Microsoft.AspNetCore.TestHost` is `5.0.3`

In .netcore 3.1 and the early .NET5, I use a variable for manage almost all microsoft dotnet nuget packages version, something like this: <https://github.com/OpenReservation/ReservationServer/blob/dev/Directory.Build.props#L18>

``` xml
<PropertyGroup>
  <DotNetPackageVersion>5.0.0</DotNetPackageVersion>
</PropertyGroup>
```

If the package version are not the same, the versions are a little in mess for me, and could not be maintained by a variable or something like this.

**Comment by paulmilbank** [comment]
Hi
I'm trying to install dotnet core and asp.net core 2.1 on Debian 9 in a docker container with
```
apt-get install -y aspnetcore-runtime-2.1=2.1.25-1
```
This fails to be found. 

Apt-cache agrees:
```
root@bfb7ddbaa0e5:/# apt-cache madison aspnetcore-runtime-2.1
aspnetcore-runtime-2.1 |   2.1.24-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.23-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.22-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.21-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.20-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.19-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.18-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.17-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.16-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.15-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.14-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.13-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.12-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.11-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |   2.1.10-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.9-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.8-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.7-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.6-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.5-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.4-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.3-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.2-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.1-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
aspnetcore-runtime-2.1 |    2.1.0-1 | https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages
```
The package list at https://packages.microsoft.com/debian/9/prod/dists/stretch/main/binary-amd64/
and the aspnet core packages at https://packages.microsoft.com/debian/9/prod/pool/main/a/aspnetcore-runtime-2.1/ and https://packages.microsoft.com/debian/9/prod/pool/main/d/dotnet-runtime-2.1/ have no updated packages either. 

Can I please get a status of when the Debian 9 / Stretch packages will be published?

**Comment by rbhanda** [comment]
Hello @paulmilbank , I have republished these packages and they should be available now.

**Comment by paulmilbank** [comment]
Thanks heaps for the quick resolution @rbhanda.

**Comment by amitpanwarIndia** [comment]
Why below issue is not being fixed in .NET 3.1 (being LTS version) while this is fixed in .NET 5. It works fine in version SDK 3.1.302 but after that version issues starts appearing and not fixed yet.
**The "PackTask" task returned false but did not log an error.** https://github.com/NuGet/Home/issues/9786

**Comment by XieJJ99** [comment]
We failed to download symbol files for .net core 3.1.12.
Steps:
[root@clrtest clr]# **docker run --rm  -ti mcr.microsoft.com/dotnet/sdk:3.1.406**
root@d12b1686aa16:/# dotnet tool install -g dotnet-symbol

> Tools directory '/root/.dotnet/tools' is not currently on the PATH environment variable.
> If you are using bash, you can add it to your profile by running the following command:
> 
> cat << \EOF >> ~/.bash_profile
> # Add .NET Core SDK tools
> export PATH="$PATH:/root/.dotnet/tools"
> EOF
> 
> You can add it to the current session by running the following command:
> 
> export PATH="$PATH:/root/.dotnet/tools"
> 
> You can invoke the tool using the following command: dotnet-symbol
> Tool 'dotnet-symbol' (version '1.0.155902') was successfully installed.
> 

root@d12b1686aa16:/# **export PATH="$PATH:/root/.dotnet/tools"**
root@d12b1686aa16:/# **dotnet symbol --symbols /usr/share/dotnet/shared/Microsoft.NETCore.App/3.1.12/libcoreclrtraceptprovider.so**

> Downloading from http://msdl.microsoft.com/download/symbols/
> ERROR: HttpSymbolStore: 404 Not Found 'http://msdl.microsoft.com/download/symbols/_.debug%2Felf-buildid-sym-04afc24fd972e821df154a37f39b565d90e5d019%2F_.debug'

**Comment by diana200** [comment]
Error on installing package for Visual Studio 2019. Processor was not recognised: 
Intel(R) Core(TM) i7-4770S CPU @ 3.10GHz   3.10 GHz

[16B4:26DC][2021-03-01T11:48:38]i305: Verified acquired payload: AspNetCoreSharedFramework_arm64 at path: C:\ProgramData\Package Cache\.unverified\AspNetCoreSharedFramework_arm64, moving to: C:\ProgramData\Package Cache\{8DB7A5DD-24AE-38E4-A117-E678460DFF34}v5.0.3.21073\aspnetcore-runtime-5.0.3-servicing.21073.3-win-arm64.msi.
[16B4:0630][2021-03-01T11:48:38]e000: Error 0x80070661: Failed to install MSI package.
[16B4:0630][2021-03-01T11:48:38]e000: Error 0x80070661: Failed to execute MSI package.
[1238:4FD4][2021-03-01T11:48:38]e000: Error 0x80070661: Failed to configure per-machine MSI package.
[1238:4FD4][2021-03-01T11:48:38]i319: Applied execute package: dotnet_runtime_5.0.3_win_arm64.msi, result: 0x80070661, restart: None
[1238:4FD4][2021-03-01T11:48:38]e000: Error 0x80070661: Failed to execute MSI package.
[16B4:0630][2021-03-01T11:48:38]i318: Skipped rollback of package: dotnet_runtime_5.0.3_win_arm64.msi, action: Uninstall, already: Absent
[1238:4FD4][2021-03-01T11:48:38]i319: Applied rollback package: dotnet_runtime_5.0.3_win_arm64.msi, result: 0x0, restart: None
